doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,642 @@
+  * Dynamic SIT LIT switching support that allows the application to switch between these modes, as long as the requirements for these modes are met.
+    You can enable this using the :kconfig:option:`CONFIG_CHIP_ICD_DSLS_SUPPORT` Kconfig option.
+  * The Kconfig option :kconfig:option:`CONFIG_CHIP_ICD_SIT_SLOW_POLL_LIMIT` to limit the slow polling interval value for the device while it is in the SIT mode.
+    You can use this to limit the slow poll interval for the ICD LIT device while it is temporarily working in the SIT mode.
+  * New device types:
+
+    * Water heater
+    * Solar power
+    * Battery storage
+    * Heat pump
+    * Mounted on/off control
+    * Mounted dimmable load control
+
+* Updated:
+
+  * Thermostat cluster with support for scheduling and preset modes, like vacation, and home or away settings.
+  * Electric Vehicle Supply Equipment (EVSE) with support for user-defined charging preferences, like specifying the time when the car will be charged.
+  * Occupancy sensing cluster with features like radar, vision, and ambient sensing.
+  * Intermittently Connected Devices feature with enhancements for the Long Idle Time (LIT) and Check-In protocol.
+    With these enhancements, the state of this feature is changed from provisional to certifiable.
+
+Thread
+------
+
+* Added Kconfig options for configuring the MLE child update timeout, child supervision interval, and child supervision check timeout.
+
+Zigbee
+------
+
+* Updated:
+
+  * :ref:`nrfxlib:zboss` to v3.11.7.0 and platform v5.1.7 (``v3.11.6.0+5.1.7``).
+    They contain several fixes related to malfunctioning in a heavy traffic environment and more.
+    For details, see :ref:`zboss_changelog`.
+  * The :ref:`ZBOSS Network Co-processor Host <ug_zigbee_tools_ncp_host>` package to the new version v2.2.5.
+
+Wi-Fi
+-----
+
+* Updated the :ref:`wifi_regulatory_channel_rules` for some countries in the :ref:`ug_nrf70_developing_regulatory_support` documentation.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+Asset Tracker v2
+----------------
+
+* Updated the Wi-Fi configurations to reduce the RAM usage by about 25 kB for an nRF91 Series DK and 12 kB for the Thingy:91 X.
+
+Connectivity bridge
+-------------------
+
+* Updated the handling of USB CDC ACM baud rate requests to make sure the baud rate is set correctly when the host requests a change.
+  This fixes an issue when using GNU screen with the Thingy:91 X.
+
+Matter bridge
+-------------
+
+* Added:
+
+  * Support for the ``UniqueID`` attribute in the Bridged Device Basic Information cluster.
+  * Version 2 of the bridged device data scheme containing ``UniqueID``.
+  * Kconfig options :ref:`CONFIG_BRIDGE_MIGRATE_PRE_2_7_0 <CONFIG_BRIDGE_MIGRATE_PRE_2_7_0>` and :ref:`CONFIG_BRIDGE_MIGRATE_VERSION_1 <CONFIG_BRIDGE_MIGRATE_VERSION_1>` to enable migration from older data schemes.
+
+nRF5340 Audio
+-------------
+
```

Comment:
Please add:

* OCT-2304: Calculation in audio_datapath. num_blks_in_fifo now considers wrapping.
* OCT-3251: Fix for nRF Auraconfig. Data was only sent on last BIS if the SD card was not used.

Reply:
Aren't these known issues?

Reply:
And if just fixes, at least the nRF Auraconfig might better fit under Bluetooth samples.
The first one might be something like:
* Updated the calculation in ``audio_datapath.num_blks_in_fifo`` to consider wrapping.

Reply:
They are resolved issues, if you want them here.
And yes, the Auraconfig should be under Bluetooth samples. My mistake.

Reply:
Updated now. First one under nRF5340 Audio and the other one under BT samples.